cmd/kube-scheduler/app/server.go

```diff
@@ -129,10 +129,6 @@ func runCommand(cmd *cobra.Command, opts *options.Options, registryOptions ...Op
 	}
 
 	if len(opts.WriteConfigTo) > 0 {
-		if err := options.WriteConfigFile(opts.WriteConfigTo, &cc.ComponentConfig); err != nil {
```

Comment:
Add a comment to say where this parameter was handled.... but maybe we can do `os.Exit(0)` in `LogOrWriteConfig`?

Reply:
Added an `os.Exit(0)` to LogOrWriteConfig